css-two-column-layout/styles.css

```diff
@@ -0,0 +1,95 @@
+h1 {
+  text-transform: uppercase;
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 50px;
+  margin: default;
+  letter-spacing: 0.3rem;
+}
+
+h3 {
+  text-transform: uppercase;
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  margin: 0;
+}
+
+p {
+  margin: 10px 0;
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 1.5;
+  color: grey;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 75%;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.card {
+  flex-basis: 50%;
+  display: flex;
+
+  /* background-color: lightblue; */
+  padding: 0 25px;
+  height: 25vh;
+}
+
+.vcenter {
+  align-items: center;
+}
+
+.wrap {
+  flex-wrap: wrap;
+}
+
+.hcenter {
+  justify-content: center;
+}
+
+.txt-center {
+  text-align: center;
+}
+
+.column-one-fourth {
+  align-self: flex-start;
+  flex-basis: 25%;
+}
+
+.column-three-fourths {
+  flex-basis: 75%;
+  padding: 40px 0
```

Comment:
The only styling that should be applied to a `column` class is either `width` or `flex-basis`. If you need additional styling, it should be applied to a separate class that you can add in addition to the `column` class.